interop-future/jvm/src/main/scala/scalaz/zio/interop/Future.scala

```diff
@@ -0,0 +1,209 @@
+package scalaz.zio
+
+import scalaz.zio.internal.impls.Env
+import scala.concurrent.ExecutionContext
+import scala.util.{ Failure, Success, Try }
+import scala.reflect.ClassTag
+
+package object future {
+
+  /**
+   * A [[scala.concurrent.Future]] is a running computation, and corresponds
+   * most closely to ZIO `Fiber`.
+   */
+  type Future[+A] = Fiber[Throwable, A]
+
+  private val Global = ExecutionContext.Implicits.global
+
+  private final def unsafeRun[E, A](ec: ExecutionContext, io: IO[E, A]): A =
+    Env.fromExecutionContext(ec).unsafeRun(io)
+
+  private final def toTry[A](e: Either[Throwable, A]): Try[A] =
+    e.fold(Failure(_), Success(_))
+
+  /**
+   * An API-compatible implementation of [[scala.concurrent.Future]], which
+   * is backed by ZIO. While this structure is not performant, due to emulation
+   * of the `Future` API, it can be useful to help migrate legacy code away
+   * from `Future` and to ZIO.
+   * */
+  object Future {
+    final val never: Future[Nothing] = Fiber.never
+
+    final val unit: Future[Unit] = Fiber.unit
+
+    final def failed[T](exception: Throwable): Future[T] =
+      Fiber.fail(exception)
+
+    final def successful[T](result: T): Future[T] =
+      Fiber.succeedLazy(result)
+
+    final def fromTry[T](result: Try[T]): Future[T] =
+      result match {
```

Comment:
You can use `fold` too

Reply:
Unfortunately it doesn't exist on Scala 2.11.